corporate/views.py

```diff
@@ -39,11 +43,14 @@ def stripe_webhook(request):
         logger.exception("Invalid signature")
         return HttpResponse(status=400)
 
-    # Handle the event
+    # Handle events
     if event.type == "checkout.session.completed":
-        pass
+        session = event["data"]["object"]
+        customer_uuid = session.metadata.customer_uuid
+        customer = Customer.objects.get(uuid=customer_uuid)
```

Comment:
Can you add Customer.objects.get_by_uuid (if it doesn't already exist) and write a unit test for it?